src/components/transactions/AssetInput.tsx

```diff
@@ -146,6 +147,18 @@ export const AssetInput = <T extends Asset = Asset>({
 
           {!onSelect || assets.length === 1 ? (
             <Box sx={{ display: 'inline-flex', alignItems: 'center' }}>
+              {value !== '' && (
+                <Button
```

Comment:
I think `IconButton` would be better to use here. That'll give us the circle hover/ripple effect, instead of the square.

Reply:
Fixed - now using `IconButton`